code/modules/language/language_holder.dm

```diff
@@ -330,8 +330,6 @@ Key procs
 								/datum/language/moffic = list(LANGUAGE_ATOM),
 								/datum/language/calcic = list(LANGUAGE_ATOM),
 								/datum/language/voltaic = list(LANGUAGE_ATOM),
-								/datum/language/nekomimetic = list(LANGUAGE_ATOM),
```

Comment:
I believe there should be a honk comment if you're removing nekomimetic since it's from upstream

Reply:
but nekomimetic is cat language? we have felinids

Reply:
ah so neko-jibberish did get pulled out too during the revert too. easy enough to fix back